robot/rover/MobilePlatform/dev/path_send_testCMakeLists.txt

```diff
@@ -0,0 +1 @@
+fuck you
```

Comment:
can you remove the fuck yous and maybe put something different or just remove the file